validator/db/src/sql/contenders.py

```
@@ -102,85 +102,84 @@ async def migrate_contenders_to_contender_history(connection: Connection) -> Non
     await connection.execute(f"DELETE FROM {dcst.CONTENDERS_TABLE}")
 
 
-async def get_contenders_for_synthetic_task(psql_db: PSQLDB, task: str, top_x: int = 5 )-> list[Contender]:
-    async with await psql_db.connection() as connection:
-        rows = await connection.fetch(
+async def get_contenders_for_synthetic_task(connection: Connection, task: str, top_x: int = 5 )-> list[Contender]:
```

Whats changed?

---

nothing except passing connection instead of psql db